source/utility.cc

```diff
@@ -15,6 +15,8 @@ void build_preconditioner
  const bool                                        higher_order_elements,
  const bool                                        symmetric)
 {
+  (void) symmetric;
+
```

Comment:
Is this a hot fix? Most likely I am mistaken but I thought the `(void)` casting renders the variable useless and thus leading to errors at line [63](https://github.com/j507/RotatingMHD/blob/eed4c1dfc21dcbd4e6db0830bafd73d4fc25700e/source/utility.cc#L63) and [82](https://github.com/j507/RotatingMHD/blob/eed4c1dfc21dcbd4e6db0830bafd73d4fc25700e/source/utility.cc#L82) when using PETSC.

Reply:
On my machine, the library compiles with additional when switching to PETSc. However, in my case, the applications are not executable when PETSc is used because of other errors.

I don't think that the `(void)`  statement does any harm, see https://godbolt.org/z/EzfqczKvG.

Reply:
This should only fix the compile time warnings when using Trilinos. PETSc is a totally different story in my opinion.